src/graphql/admin/types/payload/wallet-details.ts

```diff
@@ -0,0 +1,17 @@
+import { GT } from "@graphql/index"
+import IError from "@graphql/types/abstract/error"
+import Wallet from "@graphql/types/abstract/wallet"
+
+const WalletDetailsPayload = GT.Object({
+  name: "WalletDetailsPayload",
+  fields: () => ({
+    errors: {
+      type: GT.NonNullList(IError),
+    },
+    walletDetails: {
+      type: GT.List(Wallet),
```

Comment:
shouldn't be this a non null list too? wallet details always have a value `const walletDetails: Wallet[] = []`